src/vis/sidebar/FilterButtons.tsx

```diff
@@ -5,24 +5,47 @@ interface FilterButtonsProps {
   callback: (s: EFilterOptions) => void;
 }
 
+interface IFilterDesc {
+  name: EFilterOptions;
+  tooltip: string;
+}
+
 export function FilterButtons({ callback }: FilterButtonsProps) {
-  const options = [EFilterOptions.IN, EFilterOptions.OUT, EFilterOptions.CLEAR];
+  const options: IFilterDesc[] = [
+    {
+      name: EFilterOptions.IN,
+      tooltip: 'Filters any point not currently selected',
+    },
+    {
+      name: EFilterOptions.OUT,
+      tooltip: 'Filters all currently selected points',
+    },
+    {
+      name: EFilterOptions.CLEAR,
+      tooltip: 'Removes any existing filter',
+    },
```

Comment:
Please extract language strings using i18n.

I'm not sure though how you would make this work with enums, since you directly use the enum value as name. You might need to create separate language strings here.

Reply:
I also saw that there are still some language strings (especially the enum values) in other places that should of course be extracted, too. If you want to extract the language strings all together in a separate PR, please open a new issue + PR. Thanks.